test/python/providers/test_backend_v2.py

```diff
@@ -165,3 +167,8 @@ def test_transpile_mumbai_target(self):
         expected.measure(qr[0], cr[0])
         expected.measure(qr[1], cr[1])
         self.assertEqual(expected, tqc)
+
+    def test_transpile_parse_inst_map(self):
+        """Test that transpiler._parse_inst_map() supports BackendV2."""
+        inst_map = _parse_inst_map(inst_map=None, backend=self.backend, num_circuits=1)[0]
```

Comment:
What does this `InstructionScheduleMap` look like? I didn't think that `FakeBackendV2` has any pulse calibrations set on the instructions. I think as a first pass of a test this is fine, but after #7643 merges we should add a test with `FakeArmonk` to verify the `InstructionScheduleMap` is correct and that the transpiler is actually using it when needed.

Reply:
Created an issue here: https://github.com/Qiskit/qiskit-terra/issues/7783